src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitService.java

```diff
@@ -140,8 +151,9 @@ public Page<FaultResult> getFaultResultsPage(UUID resultUuid, FaultResultsMode m
         if (result.isPresent()) {
             Optional<Page<FaultResultEntity>> faultResultEntitiesPage = Optional.empty();
             switch (mode) {
+                case BASIC:
                 case FULL:
-                    faultResultEntitiesPage = resultRepository.findFaultResultsPage(result.get(), pageable);
+                    faultResultEntitiesPage = resultRepository.findFaultResultsPage(result.get(), pageable, mode);
                     break;
                 case WITH_LIMIT_VIOLATIONS:
                     faultResultEntitiesPage = resultRepository.findFaultResultsWithLimitViolationsPage(result.get(), pageable);
```

Comment:
The only mode possible is `WITH_LIMIT_VIOLATIONS` so it won't change anything.
I added the mode in `findFaultResultsPage` because it can be both `BASIC` or `FULL`, in order to mutualize code.